documentation/how-to-guides/deploy_magma_access_gateway.md

```diff
@@ -0,0 +1,63 @@
+# How-to: Deploy Magma Access Gateway using snap
+
+The goal of this document is to guide the operator through the process of deploying 
+Magma Access Gateway using [magma-access-gateway](https://snapcraft.io/magma-access-gateway) snap.
```

Comment:
```suggestion
Magma Access Gateway using [snap](https://snapcraft.io/magma-access-gateway).
```

Reply:
I think there are too many instances of the word "access gateway". Sorry for nitpicking.